reliability/single-fit/rum/src/test/kotlin/com/datadog/android/rum/integration/ViewLoadingTimeMetricsTests.kt

```diff
@@ -649,22 +665,785 @@ class ViewLoadingTimeMetricsTests {
+        assertThat(eventsWritten)
+            .hasSize(6)
+            .hasRumEvent(index = 0) {
+                // Initial previous view
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(previousViewKey)
+                hasViewName(previousViewName)
+                doesNotHaveInteractionToNextViewTime()
+                doesNotHaveField("feature_flag")
+            }
+            .hasRumEvent(index = 1) {
+                // Action event
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("action")
+                hasActionName(lastInteractionName)
+                hasViewName(previousViewName)
+            }
+            .hasRumEvent(index = 2) {
+                // View updated with event
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(previousViewKey)
+                hasViewName(previousViewName)
+                doesNotHaveInteractionToNextViewTime()
+                hasActionCount(1)
+            }
+            .hasRumEvent(index = 3) {
+                // Previous view stopped
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(previousViewKey)
+                doesNotHaveInteractionToNextViewTime()
+                hasActionCount(1)
+                hasViewName(previousViewName)
+            }
+            .hasRumEvent(index = 4) {
+                // New View Event
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(viewKey)
+                hasViewName(viewName)
+                doesNotHaveInteractionToNextViewTime()
+            }
+            .hasRumEvent(index = 5) {
+                // View stopped
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(viewKey)
+                doesNotHaveInteractionToNextViewTime()
+                hasViewName(viewName)
+            }
+    }
+
+    @RepeatedTest(10)
```

Comment:
I think we should remove @RepeatedTest when pushing to remote

Reply:
We have them everywhere in all the tests just to make sure we have enough test iterations there. They are super fast.